rules_building_block/collection_linux_suspicious_clipboard_activity.toml

```
@@ -0,0 +1,52 @@
+[metadata]
+creation_date = "2023/07/27"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "Multiple field support in the New Terms rule type was added in Elastic 8.6"
+min_stack_version = "8.6.0"
+updated_date = "2023/07/27"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule monitors for the usage of the most common clipboard utilities on unix systems by an uncommon process group 
+leader. Adversaries may collect data stored in the clipboard from users copying information within or between 
+applications. 
+"""
+from = "now-119m"
+interval = "60m"
+index = ["logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "Potential Suspicious Clipboard Activity Detected"
+risk_score = 21
+rule_id = "884e87cc-c67b-4c90-a4ed-e1e24a940c82"
+severity = "low"
+tags = ["Domain: Endpoint", "OS: Linux", "Use Case: Threat Detection", "Tactic: Collection", "Rule Type: BBR"]
+type = "new_terms"
+building_block_type = "default"
+query = '''
+host.os.type : "linux" and event.action : "exec" and event.type : "start" and 
```

`event.action : "exec" and event.type : "start"` will limit to process events anyway, so lets add it for clarity

```suggestion
event.category:process and host.os.type: "linux" and event.action: "exec" and event.type: "start" and 
```